doorstop/gui/main.py

```diff
@@ -443,12 +443,10 @@ def text_extendedvalue_focusout(event):
             )
 
             # External Reference
-            widget.Label(frame, text="External Reference:").grid(
-                row=7, column=0, columnspan=3, sticky=tk.W, **kw_gp
-            )
-            widget.Entry(frame, width=width_text, textvariable=self.stringvar_ref).grid(
-                row=8, column=0, columnspan=3, **kw_gsp
+            self.create_reference_widget(frame).grid(
+                row=7, rowspan=2, column=0, columnspan=2, sticky=tk.NSEW, **kw_gp
```

Comment:
it is not a regression: it does not need to be a `columnspan=3` because here only two columns are configured:

https://github.com/jacebrowning/doorstop/pull/392/files#diff-b07e5654f6b3e8f988965d73baa74cefL401